util/linkedmap/linkedmap.go

```diff
@@ -87,6 +87,13 @@ func (lm *LinkedMap[K, V]) TailNode() *ll.Element[Pair[K, V]] {
 	return ln
 }
 
+func (lm *LinkedMap[K, V]) RemoveTail() {
+	tail := lm.list.Tail
```

Comment:
In the `LinkedMap`, there is a method called `Remove ` that takes a key as a parameter and removes the corresponding entry. Could you clarify what you mean by duplicated code? Are you suggesting removing the `RemoveTail ` method and instead using the `Remove(key)` method? Since we already have the `TailNode ` method, we can obtain the key from it and then pass it to the `Remove() ` method to delete the corresponding item.